pkgs/kde/third-party/krohnkite/default.nix

```diff
@@ -0,0 +1,56 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+  nodejs_22,
+  kwin,
+  kpackage,
+  zip,
+}:
+buildNpmPackage rec {
+  pname = "krohnkite";
+  version = "0.9.7";
+
+  src = fetchFromGitHub {
+    owner = "anametologin";
+    repo = "krohnkite";
+    rev = "refs/tags/${version}";
+    hash = "sha256-8A3zW5tK8jK9fSxYx28b8uXGsvxEoUYybU0GaMD2LNw=";
+  };
+
+  npmDepsHash = "sha256-My1goFEoZW9kFA3zb8xKPxAPXm6bypyq+ajPM8zVOHQ=";
+
+  dontWrapQtApps = true;
+
+  nodejs = nodejs_22;
+
+  nativeBuildInputs = [
+    kpackage
+    zip
+    kwin
+  ];
+
+  postPatch = ''
+    cp ${./package-lock.json} package-lock.json
+    sed -i 's/7z a -tzip/zip -r/g' Makefile
```

Comment:
Could use `substituteInPlace` for a little less ugly, but not critical probably

Reply:
done